tutorials/fundamentals/clear-array.md

```diff
@@ -0,0 +1,36 @@
+There are a few ways to clear an array in JavaScript.
+
+## array.length = 0
+
+The simplest solution is to assign the array's length to the value of 0.
+The length property is readable and writeable which makes this possible.
+
+```javascript
+let array = [1,2,3,4,5];
+array.length = 0;
+array; // []
+```
+
+## array.splice(0, array.length)
+
+A more verbose approach is to use the `splice()` method.
+This function will return a copy of the array, so if you wanted to do a reassignment this would be a good use case.
```

Comment:
Not true, `splice()` modifies the array in place: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice

Reply:
Right, but if you `console.log(array.splice(0, array.length)` It returns the array before it deletes everything. After that function call `console.log(array)` is empty. My mistake for not making that clearer. I guess I could have said, "A copy of the array before deleting the entries"?